hardhat.config.js

```diff
@@ -8,6 +8,7 @@ require('@openzeppelin/hardhat-upgrades');
  */
 
 const HARMONY_PRIVATE_KEY = process.env.PRIVATE_KEY;
+const PROJECT_ID = process.env.INFURA_PROJECT_ID
```

Comment:
Infura sometimes goes down for Ethereum testnets - I suggest we allow freeform provider URLs